packages/relay/src/lib/eth.ts

```diff
@@ -56,6 +57,7 @@ import { IDebugService } from './services/debugService/IDebugService';
 import { DebugService } from './services/debugService';
 import { IFeeHistory } from './types/IFeeHistory';
 import { ITransactionReceipt } from './types/ITransactionReceipt';
+import { Trie } from '@ethereumjs/trie';
```

Comment:
My editor says it cannot find this module

<img width="882" alt="image" src="https://github.com/user-attachments/assets/2ca5b065-5d0f-42ba-a8b4-e4d34992edb6">

Reply:
You need to install the Trie library with `npm install`